index.html

```diff
@@ -273,20 +284,22 @@ <h2 class="connect">
 
     <!-- Section, contains contact form -->
     <section class="section-contact">
-        <h1 class="title contact-title">
-            Contact me
-        </h1>
-        <p class="contact-content">
-            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
-            I’d love to help with it!
-        </p>
-        <div class="contact-form">
-            <form action="https://formspree.io/f/xpzebrvw" method="POST">
-                <input type="text" name="name" maxlength="30" placeholder="Lorem Ipsum" required>
-                <input type="email" name="email" placeholder="loremipsum@gmail.com" required>
-                <textarea name="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here..." required></textarea>
-                <button class="project-link" type="submit">Get in touch</button>
-            </form>
+        <div>
+            <h1 class="title contact-title">
+                Contact me
+            </h1>
+            <p class="contact-content">
+                If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                I’d love to help with it!
+            </p>
+            <div class="contact-form">
+                <form action="https://formspree.io/f/xpzebrvw" method="POST">
+                    <input type="text" name="name" maxlength="30" placeholder="Lorem Ipsum" required>
+                    <input type="email" name="email" placeholder="loremipsum@gmail.com" required>
+                    <textarea name="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here..." required></textarea>
+                    <button class="project-link" type="submit">Get in touch</button>
+                </form>
+            </div>
         </div>
     </section>
```

Comment:
- [ ] As mentioned previously pelase make sure the background image for the "Contact me" section is aligned properly as shown in the design: 
- Your design:
  ![image](https://user-images.githubusercontent.com/104968236/236421183-92a516f1-9bc9-4b98-a6b4-b54cd8f67d2c.png)
- Figma design:
  ![image](https://user-images.githubusercontent.com/104968236/236421346-25978d6c-14e6-48b5-b39b-ee49c99ed991.png)

- [ ] Also please make sure to remove any section descriptions from your readme as they're only there to guide the author on what to place on each section: 👇
    ![image](https://user-images.githubusercontent.com/104968236/236422430-87d0fb9e-b740-4053-9947-01190e4991f2.png)